first draft for user_info (#60)
PyDrocsid/documentation: docs/cogs/information/user_info.md

> # User Info

Reviewer: ```suggestion
# User info
```

> This cog contains some commands to show information about Discord user accounts.

Reviewer: ```suggestion
This cog contains commands to show information about Discord user accounts.
```

## `user`

> The `.user` command shows general information about a given user.

Reviewer: ```suggestion
The `.user` command shows general information about the requested user.
```

```css
.[userinfo|user|uinfo|userstats] [user]
```

|Argument|Required|Description|
|:------:|:------:|:----------|

> |`user`||The user whose information is requested. If omitted, the requesting user receives their own information via direct messages.|

Reviewer 2: ```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via as a direct message.|
```

> |`user`||The user whose information is requested. If omitted, the requesting user receives their own information via a direct message.|

Reviewer 2: ```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information as a direct message.|
```

> |`user`||The user whose information is requested. If omitted, the requesting user receives their own information via as a direct message.|

Reviewer: ```suggestion
|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via a direct message.|
```

!!! note
    - Every user is allowed to request their own information.
    - Requesting other users' information, however, requires the `user_info.view_userinfo` permission.

The user information given by this command includes:

- Current username, discriminator, avatar and id
- Moderation statistics:
    - Reports
    - Warns
    - Mutes
    - Kicks
    - Bans
- Membership status
- User activity
- Active punishments

!!! note
    Moderation statistics:

    - The term `active` refers to the number of times this user *has actively* reported/punished someone else.
    - The term `passive` refers to the number of times this user *has been* reported/punished e.g. by a moderator.

## `joined`

> The `.joined` command shows a rough estimate of how much time has passed since the last verification of a given member. If no `Verified` role has been configured yet, this estimate refers to the last join date of this member.

Reviewer 2: Keep in mind that the actual calculation is probably going to be changed soon, so either remember to update this accordingly or make the documentation more generic here

```css
.joined [member]
```

|Argument|Required|Description|
|:------:|:------:|:----------|
|`member`||The member whose information is requested. If omitted, this defaults to the requesting member.|

## `ulog`
The `.ulog` command shows the moderation log of a given user.

```css
.[userlogs|userlog|ulog] [user]
```

|Argument|Required|Description|
|:------:|:------:|:----------|

> |`user`||The user whose log is requested. If omitted, the requesting user receives their own log via direct messages.|

Reviewer 2: ```suggestion
|`user`||The user whose log is requested. If omitted, the requesting user receives their own log as a direct message.|
```

!!! note
    - Every user is allowed to request their own moderation log.
    - Requesting other users' logs, however, requires the `user_info.view_userlog` permission.

The user log given by this command contains the following events:

- Account creation
- Guild joins and leaves
- Username and nickname changes
- Verification state changes
- Discord invite approvals and removals

> - Moderation actions:

Reviewer 2: ```suggestion
- Passive moderation actions:
```
